ansible-ipi-install/roles/node-prep/tasks/45_networking_facts.yml

```diff
@@ -34,7 +34,7 @@
   set_fact:
     provisioning_subnet: "{{ ip | ipaddr('network') }}/{{ ip | ipaddr('prefix') }}"
   vars:
-    ip: "{{ ansible_facts.provisioning.ipv4.address }}/{{ansible_facts.provisioning.ipv4.netmask }}"
+    ip: "{{ ansible_facts[provisioning_bridge]['ipv4']['address'] }}/{{ ansible_facts[provisioning_bridge]['ipv4']['netmask'] }}"
```

Comment:
Nice! 

Can you apply the same logic here for the IPv6 ? (line 48)

Reply:
yes, will do, now that I was able to verify it works for ipv4 :-)

Reply:
and maybe we want that for baremetal (even though I dont need it) too while am at it?

Reply:
if you dont mind that be great yuvalk! :)

Reply:
added ipv6
but noted two (weird?) things:
- rhcos cache doesnt support ipv6?
- for ipv4 we use just 'address' but for ipv6 it uses [0]['address']

Reply:
Regarding the rhcos cache it doesn't support IPv6 yet. When I get this PR #262 this will get fixed. So I'd leave the rhcos cache part alone for now as that PR will put the right details in. 

Regarding the IPv6 ansible facts in 45_ script, I'd think that what you wrote here: 
```
ansible_facts[provisioning_bridge]['ipv6'][0]['address']
```
will work but I will test it out shortly in an IPv6 env to confirm